### PR TITLE
Update to 'Teams naming policy'

### PR DESCRIPTION
**Pull request by SamCosby**
Teams doesn't respect this feature at this time while this feature is in Private Preview for Groups. Unless you're in that Private Preview, you will not be able to do this in Teams at this time.

### CODE REVIEW

File: Teams/plan-Office-365-groups.md [hunk 1]
```diff
@@ -22,8 +23,10 @@ There are two scenarios that warrant some discussion around planning of Office 3
 
 -   Second, for customers who don’t have existing Groups defined in Office 365, you can either create them using the Office 365 admin portal, the Teams web, or desktop clients. As mentioned previously, manage all future membership to the Office 365 Group using the Teams client. Since membership to a Team is also defining membership to Office 365 Groups, you should prepare people for this change.
 
-## Teams respects Office 365 Groups naming policy
-Any Office 365 Groups naming policy that has been set by your admin will be applied in Teams when users create or edit team names. This includes things like mandatory prefixes or suffixes and excluding banned words.
+## Teams respects Office 365 Groups naming policy (Private Preview)
+Any Office 365 Groups naming policy that has been set by your admin will be applied in Teams when users create or edit team names. This includes things like mandatory prefixes or suffixes and excluding banned words. 
+
+- Note, this feature is in Private Preview, which means that if you're not in this Preview, Teams will not adhear to this yet.
```
Comment (ninadara) [inline Teams/plan-Office-365-groups.md]: Should we add the note box?

**Comment by SamCosby** [comment]
Hey Nishanth,

I think we’d need a note box, new with GitHub editing…

Thank you,

Sam

Sam Cosby
Microsoft Corporation | Support Escalation Engineer | Intelligent Communications
P: +1-857-453-6115
Schedule: 8:00AM-5:00PM EST
[http://www.onetechstop.net/wp-content/uploads/2016/10/msteamslogo.png]

**Comment by LolaJacobsen** [comment]
I updated the article and pushed live:
https://docs.microsoft.com/en-us/MicrosoftTeams/plan-office-365-groups

I'm closing this PR without accepting it - but I made the update myself.

Next time, issue your PR to Master (we never publish directly to Live - we always go through Master first. Think of Master as a staging site.)

**Comment by SamCosby** [comment]
Noted for next time.. Thanks for this Lola.

**Pull request closed without merging**